addons/account/models/ir_actions_report.py

```diff
@@ -33,3 +33,9 @@ def postprocess_pdf_report(self, record, buffer):
             if attachment:
                 attachment.register_as_main_attachment(force=False)
         return res
+
+    @api.model
+    def render_qweb_pdf(self, res_ids=None, data=None):
+        if self.model == 'account.move' and not self.env[self.model].browse(res_ids).is_invoice(include_receipts=True):
```

Comment:
It doesn't handle the case when a user tries to print several journal entries at the same time.
You can do something like that:
```python
if self.model == 'account.move' and any(not m.is_invoice(include_receipts=True) for m in self.env[self.model].browse(res_ids)```

Reply:
Done